models/Message.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const MessageSchema = Schema({
+  fromUserName: {
```

Comment:
I didn't realize that the ChatList was an array of User objects when I requested that the Messages schema have usernames of both users. I originally thought that the FE would need to be making a couple get requests to obtain the information of the other user to render out. But since that isn't the case (the User can be passed from the ChatList), these usernames are not really necessary. I'm not sure if you want to remove them. Sorry for all the fuss!

Reply:
alright sound good i'll remove them